File: account/src/main/java/serp/project/account/core/domain/dto/request/GetOrganizationParams.java
/**
 * Author: QuanTuanHuy
 * Description: Part of Serp Project
 */

package serp.project.account.core.domain.dto.request;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@SuperBuilder
public class GetOrganizationParams extends BaseGetParams {
    private String search;
    private String status;
    private String type;
}

File: api_gateway/src/core/domain/dto/request/account/get_organization_params.go
/*
Author: QuanTuanHuy
Description: Part of Serp Project
*/

package request

type GetOrganizationParams struct {
	Search           *string `json:"search,omitempty"`
	Status           *string `json:"status,omitempty"`
	OrganizationType *string `json:"type,omitempty"`
	Page             *int    `json:"page,omitempty"`
	PageSize         *int    `json:"pageSize,omitempty"`
}
